apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -451,6 +451,11 @@ def flash(self, options):
             recover_args.extend(_get_nrf_device_args(options))
             check_call(recover_args, cwd=API_SERVER_DIR / "build")
 
+        if "west_cmd" in options:
+            west_cmd = options["west_cmd"]
+            cmake_args = ["cmake", "..", f"-DWEST={west_cmd}"]
```

Comment:
i wonder if we should add this option to the original cmake invocation on line 395? i think it makes sense to have `west_cmd`, but thinking it's better to place there just in case this triggers a rebuild.

Reply:
@areusch Yeah, I was following what we had previously to Project API. Makes sense. Done in https://github.com/apache/tvm/pull/8941/commits/395bcbe88399adcd0134711ad6da56d6a42297ea

Thanks for the review!